.cloudbuild/graal-build.yaml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - name: oracle/graalvm-ce:latest
```

Comment:
It would probably be a better idea to lock to a fixed tag (say `20.2.0-java8`, which is what `latest` is currently aliased to, or `20.2.0-java11` - see https://hub.docker.com/r/oracle/graalvm-ce/tags).  You would also know at a glance what exactly you're getting.

Reply:
Done.